pkg/osutils/osutils.go

```diff
@@ -0,0 +1,125 @@
+package osutils
```

Comment:
I don't know if you want to execute it in a specific node, otherwise, this os utils will only execute inside torpedo pod, is that what you want?

Reply:
yes, this is only for torpedo pod. As I'm using in multiple places I decided to move all commands to the package.